### PR TITLE
doc, english

### PR DESCRIPTION
**Pull request by baem2**

### CODE REVIEW

File: reledmac.dtx [hunk 1]
```diff
@@ -6739,14 +6739,14 @@
...
+% To solve this problem, we use a crossref mechanism. At the first run, we put a label, but we do not write any \cs{@ref} command. When the value of the label can be tested, that is after three runs, we're doing:
 % \begin{itemize}
-%  \item If the first argument of \cs{edtext} is typeset on only one line, we first call \cs{flag@end}, at the point we normally call \cs{flag@start}, at the beginning of the content of the first argument,  and we call \cs{flag@end} at the point we normally call \cs{flag@start}, at the end of the content of the first argument.
+%  \item If the first argument of \cs{edtext} is typeset on only one line, we first call \cs{flag@end}, at the point we normally call \cs{flag@start}, at the beginning of the content of the first argument, and we call \cs{flag@end} at the point we normally call \cs{flag@start}, at the end of the content of the first argument.
 %  \item If the first argument of \cs{edtext} is typeset on only two lines, we use the normal order.
 % \end{itemize}
-% This system compensate the problem of order when writing in auxiliary files.
+% This system fixes the problem of order when writing in auxiliary files.
```
Comment (maieul) [inline reledmac.dtx]: I am not sur we can tell it fix the problem. It does not really fix, but avoid it.

**Comment by baem2** [comment]
Okay, so this should be better.

**Comment by baem2** [comment]
And some more fixes added, should be fine now.

**Comment by maieul** [comment]
Many thanks ! That is perfect.

**Pull request merged**